lib/active_record/connection_adapters/spanner/database_statements.rb

```diff
@@ -219,9 +219,8 @@ def to_types_and_params binds
           end.to_h
           params = binds.enum_for(:each_with_index).map do |bind, i|
             type = bind.respond_to?(:type) ? bind.type : ActiveModel::Type::Integer
-            value = bind
-            value = type.serialize bind.value, :dml if type.respond_to?(:serialize) && type.method(:serialize).arity < 0
-            value = type.serialize bind.value if type.respond_to?(:serialize) && type.method(:serialize).arity >= 0
+            value = ActiveRecord::Type::Spanner::SpannerActiveRecordConverter
+                    .serialize_with_transaction_isolation_level(type, bind.value, :mutation)
```

Comment:
It seems that this will always use isolation level `:mutations`. I don't think that is correct.

Reply:
`:dml` is correct, pardon the sloppy copy pasta